Assets/IntegrationTests/TestShopifyProducts.cs

```diff
@@ -46,17 +47,16 @@ public class TestShopifyProducts : MonoBehaviour {
 
             yield return waiter;
 
-            Assert.IsTrue (waiter.IsStopped, Utils.MaxQueryMessage);
+            Assert.IsTrue(waiter.IsStopped, Utils.MaxQueryMessage);
```

Comment:
This is a comment on shipped code, but I noticed `MaxQueryMessage` is wrong. As per the docs this message is shown when the test fails:

> Verifies that the specified condition is true. The assertion fails if the condition is false. Displays a message if the assertion fails.

So that message should be something like "Query failed to run in x seconds" or whatever instead of "Query finished in 1 seconds"